docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,170 @@
+---
+title: ".NET docs what's new (November 2019)"
+description: "What's new in .NET docs in November 2019."
+ms.date: 12/13/2019
+---
+
+# .NET docs what's new (November 2019)
+
+Welcome to what's new in .NET docs for November 2019. Listed below are some of the recent docs we've published.
+
+## .NET Core
+
+- [Breaking changes, version 2.2 to 3.0 - .NET Core](../core/compatibility/2.2-3.0.md)
```

Comment:
I'd remove the automated suffix from these or use the H1 like Mark suggested

Reply:
I did use the H1. I like that a lot better.